StringFinder.java

```diff
@@ -0,0 +1,38 @@
+import java.io.File;
+import java.util.Scanner;
+
+public class StringFinder {
+
+    public static void main(String[] args) {
+        if (args.length != 3) {
+            System.out.println("Wrong number of input parameters!");
+        } else {
+            if (args[0].equals("search")) {
```

Comment:
what if the input is in none strings?

Reply:
Fixed so the program can handle other types by converting them to strings

Reply:
For instance ints.